### PR TITLE
docs: minor update to readme.md

### PR DESCRIPTION
**Pull request by anupriya0703**
This change adds info about 'openebs/mayastor-extensions' repo.

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Mayastor's source code and documentation are distributed amongst a number of Git
 - [openebs/mayastor-control-plane](https://github.com/openebs/mayastor-control-plane) : contains the source code of the control plane components
 - [openebs/mayastor-api](https://github.com/openebs/mayastor-api) : contains common protocol buffer definitions and OpenAPI specifications for Mayastor components
 - [openebs/mayastor-dependencies](https://github.com/openebs/mayastor-dependencies) : contains dependencies common to the control and data plane repositories
+- [openebs/mayastor-extensions](https://github.com/openebs/mayastor-extensions) : contains components and utilities that extend core Mayastor's functionalities.
```
Comment (avishnu) [inline README.md]: "that provide extended functionalities like ease of installation, monitoring and observability aspects."

How about the above?

**Review by avishnu — APPROVED** [review]
lgtm

**Review by GlennBullingham — APPROVED** [review]
Approved

**Review by GlennBullingham — CHANGES_REQUESTED** [review]
Actually, taking that approval back ;-)

Line 31 is the only line with a full-stop at the end.  Remove it for consistency?

**Review by GlennBullingham — APPROVED** [review]
LGTM

**Pull request merged**